orcid-core/src/main/java/org/orcid/core/manager/impl/OrcidProfileManagerImpl.java

```diff
@@ -2220,32 +2220,32 @@ private void setDefaultVisibility(OrcidProfile orcidProfile) {
                 OrcidBio bio = orcidProfile.getOrcidBio();
                 if (bio.getBiography() != null) {
                     if (bio.getBiography().getVisibility() == null) {
-                        bio.getBiography().setVisibility(OrcidVisibilityDefaults.BIOGRAPHY_DEFAULT.getVisibility());
+                        bio.getBiography().setVisibility(Visibility.PRIVATE);
```

Comment:
@lpaglione @amontenegro @caoyler is this right? Bio is always going to be set to private when one is created?